Cannot /stop the server

When using the `/stop` command in console or as player you get this:

```

2016/01/24 12:38:00 [INFORMATION] CONSOLE: Stopping the server..
2016/01/24 12:38:00 [INFORMATION] The server is shutting down...
2016/01/24 12:38:00 [INFORMATION] Paulomart kicked: Server shutting down.
2016/01/24 12:38:00 [SCHWERWIEGEND] Error while executing GlowTask{id=14, plugin=null, sync=true: net.glowstone.ConsoleManager$CommandTask@377ce467}
org.bukkit.command.CommandException: Unhandled exception executing 'stop' in org.bukkit.command.defaults.StopCommand(stop)
    at org.bukkit.command.SimpleCommandMap.dispatch(SimpleCommandMap.java:192)
    at net.glowstone.GlowServer.dispatchCommand(GlowServer.java:1165)
    at net.glowstone.ConsoleManager$CommandTask.run(ConsoleManager.java:214)
    at java.util.concurrent.Executors$RunnableAdapter.call(Unknown Source)
    at java.util.concurrent.FutureTask.run(Unknown Source)
    at net.glowstone.scheduler.GlowTask.run(GlowTask.java:163)
    at net.glowstone.scheduler.GlowScheduler.pulse(GlowScheduler.java:181)
    at net.glowstone.scheduler.GlowScheduler.access$100(GlowScheduler.java:24)
    at net.glowstone.scheduler.GlowScheduler$2.run(GlowScheduler.java:110)
    at java.util.concurrent.Executors$RunnableAdapter.call(Unknown Source)
    at java.util.concurrent.FutureTask.runAndReset(Unknown Source)
    at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.access$301(Unknown Source)
    at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(Unknown Source)
    at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
    at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
    at java.lang.Thread.run(Unknown Source)
Caused by: java.util.ConcurrentModificationException
    at java.util.HashMap$HashIterator.nextNode(Unknown Source)
    at java.util.HashMap$KeyIterator.next(Unknown Source)
    at java.util.Collections$UnmodifiableCollection$1.next(Unknown Source)
    at net.glowstone.GlowServer.shutdown(GlowServer.java:648)
    at org.bukkit.Bukkit.shutdown(Bukkit.java:700)
    at org.bukkit.command.defaults.StopCommand.execute(StopCommand.java:28)
    at org.bukkit.command.SimpleCommandMap.dispatch(SimpleCommandMap.java:185)
    ... 15 more
```

--- a/src/main/java/net/glowstone/GlowServer.java
+++ b/src/main/java/net/glowstone/GlowServer.java
@@ -645,7 +645,7 @@ public final class GlowServer implements Server {
         pluginManager.clearPlugins();
 
         // Kick all players (this saves their data too)
-        for (Player player : getOnlinePlayers()) {
+        for (Player player : new ArrayList<Player>(getOnlinePlayers())) {
             player.kickPlayer(getShutdownMessage());
         }
 

